File: AutoSync/3392-count-subarrays-of-length-three-with-a-condition/solution.cpp
class Solution {
public:
    int countSubarrays(vector<int>& nums) {//普通暴力解决
        int ans = 0;
        for (int i = 0; i < nums.size()-2; i++)
        {
            if((nums[i+2] +nums[i]) *2 == nums[i+1]){//题目要求
                ans++;
            }
        }
        return ans;
    }
};

File: AutoSync/3392-count-subarrays-of-length-three-with-a-condition/solution.java
class Solution {
    public int countSubarrays(int[] nums) {
        int n = nums.length;
        int ans=0;
        for(int i=0;i<n-2;i++){
            if((nums[i+2]+nums[i])*2==nums[i+1]){
                ans++;
            }
        }
        return ans;
    }
}
